galley/docker/Dockerfile.galley

```diff
@@ -2,12 +2,15 @@
 ARG BASE_DISTRIBUTION=default
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
+# hadolint ignore=DL3006
 FROM istionightly/base_debug as default
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
-FROM gcr.io/distroless/static as distroless
+# hadolint ignore=DL3007
+FROM gcr.io/distroless/static:latest as distroless
```

Comment:
is there not a release, instead of using `latest`?  Using latest IMO is wrong, as you don't know which version of software your building with.

Reply:
@sdake : there are other tag, but they're basically just some kind of random hashes, according to what I see here: https://pantheon.corp.google.com/gcr/images/distroless/GLOBAL/static?gcrImageListsize=30

Reply:
I can't access that link (requires google SSO) however, I took a look here:
https://console.cloud.google.com/gcr/images/distroless/GLOBAL/static?gcrImageListsize=30

And it does appear that there is no specific versioning going on in distroless.  Can you open an issue in their issue tracker on this point?